packages/@lwc/engine/src/framework/wc.ts

```diff
@@ -5,42 +5,21 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 import { ComponentConstructor } from './component';
-import { isUndefined, isObject, isNull, getOwnPropertyNames, ArrayMap } from '../shared/language';
+import { isUndefined, getOwnPropertyNames, ArrayMap } from '../shared/language';
 import { createVM, appendRootVM, removeRootVM, getCustomElementVM, CreateVMInit } from './vm';
 import { EmptyObject } from './utils';
 import { getComponentDef } from './def';
 import { getPropNameFromAttrName, isAttributeLocked } from './attributes';
 import { HTMLElementConstructor } from './base-bridge-element';
 import { patchCustomElementWithRestrictions } from './restrictions';
 
-/**
- * This function builds a Web Component class from a LWC constructor
- * so it can be registered as a new element via customElements.define()
- * at any given time. E.g.:
- *
- *      import { buildCustomElementConstructor } from 'lwc';
- *      import Foo from 'ns/foo';
- *      const WC = buildCustomElementConstructor(Foo);
- *      customElements.define('x-foo', Foo);
- *      const elm = document.createElement('x-foo');
- *
- */
-export function buildCustomElementConstructor(
-    Ctor: ComponentConstructor,
-    options?: ShadowRootInit
```

Comment:
removing the possibility to use `options` since internally we don't expose that anymore.